aptos/contracts/Move.toml

```diff
@@ -2,11 +2,13 @@
 name = "Aptos"
 version = "0.0.1"
 
+#TODO: pin versions before mainnet release
 [dependencies]
 AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-framework/", rev = "devnet" }
 MoveStdlib = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/move-stdlib/", rev = "devnet" }
 AptosStdlib = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-stdlib/", rev = "devnet" }
 AptosToken = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-token/", rev = "devnet" }
+U256 = { git = "https://github.com/pontem-network/u256", rev = "main"  }
```

Comment:
Added a dependency here, but it breaks `move-analyzer` unfortunately. Might just vendor the source file (it's apache 2)